example/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -5,7 +5,7 @@
 import FlutterMacOS
 import Foundation
 
-import app_links_macos
+import app_links
```

Comment:
Do we need these changes on the generated files for macos and windows?

Reply:
That was accidentally committed, reverting that!

Reply:
Done